Framework/PythonInterface/plugins/algorithms/ReflectometrySliceEventWorkspace.py

```diff
@@ -38,6 +38,16 @@ def PyInit(self):
                              doc='Group name for the output workspace(s).')
         self.declareProperty("UseNewFilterAlgorithm", True, doc='If true, use the new FilterEvents algorithm instead of FilterByTime.')
 
+    def validateInputs(self):
+        issues = {}
+        workspace = self.getProperty("InputWorkspace").value
+        # Skip check for workspace groups
```

Comment:
This comment is really helpful. Would it be worth adding a brief explanation of why we're skipping workspace groups?

Reply:
> Code changes look good and worked well when I tested the script. I got a file not found error when trying to test the GUI with run 66718 - is there something I need to set up on my end to get that to work?

It will need the data archive - in Manage User Directories, ensure the drop-down for Search Data Archive is enabled (I usually set it to 'default facility only')

Reply:
> This comment is really helpful. Would it be worth adding a brief explanation of why we're skipping workspace groups?

It's common practice to skip checks on workspace groups so it's probably not necessary

Reply:
Thanks, enabling the Search Data Archive got that working.